helpers/__tests__/request-test.ts

```diff
@@ -59,6 +65,28 @@ describe('parseError', () => {
   });
 });
 
+describe('parseJSON', () => {
+  it('should return a json response', () => {
+    const body = { test: 2 };
+    const response = new Response(JSON.stringify(body), { status: 200 });
+    const json = jest.spyOn(response, 'json');
+    const jsonResponse = parseJSON(response);
+    expect(json).toBeCalled();
+    return expect(jsonResponse).resolves.toEqual(body);
+  });
+});
+
+describe('parseText', () => {
+  it('should return a json response', () => {
+    const body = 'test';
+    const response = new Response(body, { status: 200 });
+    const json = jest.spyOn(response, 'text');
+    const textResponse = parseText(response);
+    expect(json).toBeCalled();
```

Comment:
I would rename ``` json ``` by ``` text ```. It confused me for 5 seconds.

Reply:
Ah yes indeed, copy-pasta mistake